roles/idrac_gather_facts/tests/asserts/controller_assert.yml

```diff
@@ -10,30 +11,60 @@
     body_format: "{{ idrac_gather_facts_uri_body_format }}"
     status_code: "{{ idrac_gather_facts_uri_status_code }}"
     return_content: "{{ idrac_gather_facts_uri_return_content }}"
-  register: controller_result
+  register: storage_entity
   no_log: true
 
-- name: Set controller facts
+- name: Get storage entities
+  ansible.builtin.set_fact:
+    collected_storage_entity:
+      "{{ (collected_storage_entity | default([])) + [item['@odata.id']] }}"
+  loop: "{{ storage_entity.json.Members }}"
+
+- name: Get Controllers URL information.
+  ansible.builtin.uri:
+    url: "https://{{ hostname }}{{ item }}"
+    <<: *uri_params
+  loop: "{{ collected_storage_entity }}"
+  register: controllers_url
```

Comment:
can you rename this register var to idrac_gather_facts_

Reply:
Done